python/ray/experimental/state/state_cli.py

```diff
@@ -669,3 +677,321 @@ def object_summary(ctx, timeout: float, address: str):
             ),
         )
     )
+
+
+@click.group("logs", invoke_without_command=True)
+@click.pass_context
+def logs_state_cli_group(ctx):
+    """Get logs based on filename (file) or resource identifiers (actor)
+
+    Usage:
+
+        Get all the log files available on a node (ray address could be
+        obtained from `ray start --head` or `ray.init()`)
+
+        ```
+        ray log --address=<ray address>
+        ```
+
+        [ray logs file] Print the last 500 lines of raylet.out on a head node.
+
+        ```
+        ray logs file raylet.out --tail 500
```

Comment:
Is it possible to do `ray logs raylet.out` too? If so, can you mention `ray logs` is a alias of `ray logs file`?